db/migrate/20200411180309_add_users_to_orders.rb

```diff
@@ -0,0 +1,5 @@
+class AddUsersToOrders < ActiveRecord::Migration[5.1]
+  def change
```

Comment:
had to create user order one to many relationship here to be able to get a users orders